compiler/src/test/scala/org/scalajs/core/compiler/test/util/DirectTest.scala

```diff
@@ -26,7 +28,7 @@ abstract class DirectTest {
         List(
             "-d", testOutputPath,
             "-bootclasspath", scalaLibPath,
-            "-classpath", scalaJSLibPath) ++
+            "-classpath", classpath.mkString(":")) ++
```

Comment:
This doesn't work on Windows (I tested). You need to use `File.pathSeparator` (I tested that it fixes the issue).